docs-svelte-kit/src/app.css

```diff
@@ -6,7 +6,7 @@
   --font-mono: "Fira Mono", monospace;
   --pure-white: #ffffff;
   --primary-color: #b9c6d2;
-  --secondary-color: #d0dde9;
+  --secondary-color: #676778;
```

Comment:
`--secondary-color` is not used at all. So I updated this color according to Svelte site7s color.